.yamllint.yaml

```diff
@@ -11,6 +11,11 @@ yaml-files:
   - OWNERS
 
 rules:
+  braces:
+    min-spaces-inside: 1
+    max-spaces-inside: 1
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 0
```

Comment:
The only functional change to this PR was adding this block, which makes yamllint agree with yamlfmt on the formatting of the braces in `examples/site.yml`